ircd/files/ircd.yaml

```diff
@@ -268,7 +273,7 @@ server:
         enabled-for-always-on: true
 
         # fake TLD at the end of the hostname, e.g., pwbs2ui4377257x8.oragono
-        netname: "hashbang.sh"
+        netname: "irc"
```

Comment:
Why `irc` rather than something even shorter?

Reply:
I think, e.g., `c9un4fx3he27q.irc` strikes a good balance in terms of making it clear that this is a "pseudo-hostname". (It helps that most of the original TLDs are three letters.)